### PR TITLE
added VCF overview

### PR DESCRIPTION
**Pull request by ekiernan**
Added description of Illumina Genotyping Array Pipeline VCF to WARP docs

### CODE REVIEW

File: website/docs/.vuepress/config.js [hunk 1]
```diff
@@ -128,7 +128,8 @@ module.exports = {
               title: "Illumina Genotyping Arrays Pipeline",
               collapsable: true,
               children: [
-                "Pipelines/Illumina_Genotyping_Arrays_Pipeline/IlluminaGenotypingArray.documentation"
+                "Pipelines/Illumina_Genotyping_Arrays_Pipeline/IlluminaGenotypingArray.documentation",
+                "Pipelines/Illumina_Genotyping_Arrays_Pipeline/Illumina_array_spec"
```
Comment (gbggrant) [inline website/docs/.vuepress/config.js:132]: I think I'd call it Illumina_genotyping_array_spec, just to be consistent.

File: website/docs/documentation/Pipelines/Illumina_Genotyping_Arrays_Pipeline/IlluminaGenotypingArray.documentation.md [hunk 2]
```diff
@@ -2,20 +2,20 @@
 
 | Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
 | :----: | :---: | :----: | :--------------: |
-| [Version 1.11.0](https://github.com/broadinstitute/warp/releases) | Oct 1, 2020 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+| [Version 1.11.0](https://github.com/broadinstitute/warp/releases) | April 1, 2021 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
```
Comment (gbggrant) [inline website/docs/documentation/Pipelines/Illumina_Genotyping_Arrays_Pipeline/IlluminaGenotypingArray.documentation.md:5]:
```suggestion
| [Version 1.11.0](https://github.com/broadinstitute/warp/releases) | May 1, 2021 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
```
?  Or is that the latest release date.

File: website/docs/documentation/Pipelines/Illumina_Genotyping_Arrays_Pipeline/Illumina_array_spec.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+1. #CHROM 
+2. POS 
+3. ID 
+4. REF 
+5. ALT 
+6. QUAL 
+7. FILTER 
+8. INFO
+
+These columns are standard to each VCF, but the Illumina Genotyping Array Pipeline has fields with additional attributes as described below:
```
Comment (gbggrant) [inline website/docs/documentation/Pipelines/Illumina_Genotyping_Arrays_Pipeline/Illumina_array_spec.md:45]: Actually, the attributes below are not separate fields, they are values that can be found in the particular column.  For instance the Filter column may contain the fields/attributes listed in the next section.

**Review by gbggrant — APPROVED** [review]

**Review by jessicaway — APPROVED** [review]
Looks good, just 1 minor comment

File: website/docs/documentation/Pipelines/Illumina_Genotyping_Arrays_Pipeline/Illumina_array_spec.md (hunk 3 above)
Comment (ekiernan) [reply website/docs/documentation/Pipelines/Illumina_Genotyping_Arrays_Pipeline/Illumina_array_spec.md:45]: @gbggrant Thanks for clarifying. Will update this and also add that pipeline VCS has an additional format column.

File: website/docs/.vuepress/config.js (hunk 1 above)
Comment (ekiernan) [reply website/docs/.vuepress/config.js:132]: Updated this!

File: website/docs/documentation/Pipelines/Illumina_Genotyping_Arrays_Pipeline/Illumina_array_spec.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
+# VCF Overview: Illumina Genotyping Array 
+
+The [Illumina Genotyping Array Pipeline](https://github.com/broadinstitute/warp/blob/develop/pipelines/broad/genotyping/illumina/IlluminaGenotypingArray.wdl) v1.11.0 pipeline produces a VCF (variant calling format) output with data processing and sample-specific genotype information. The VCF follows the format listed in the VCF 4.2 specification, but additionally contains fields and attributes that are unique to the Arrays pipeline.
```
[the remark this answers is not in the document]
Comment (ekiernan) [reply website/docs/documentation/Pipelines/Illumina_Genotyping_Arrays_Pipeline/Illumina_array_spec.md]: Yes, it should be linked. Will update. Thank you!

**Comment by ekiernan** [comment]
@jessicaway @gbggrant Since it's failing pipeline release, I can't merge (not an admin). Do you have the ability to merge?

**Comment by jessicaway** [comment]
Branch protection has to be edited to allow us to merge when a required status is failing

**Pull request merged**